scripts/rebalance_subject_split.py

```
@@ -0,0 +1,283 @@
+        if not isinstance(sample_id, str) or not sample_id:
+            continue
+        present = target_labels_in_row(row, label_set)
+        if not present:
+            continue
+        eval_candidates.append(EvalCandidate(sample_id=sample_id, labels=present))
+        for label in present:
+            eval_counts[label] += 1
+
+    totals = {label: train_counts[label] + eval_counts[label] for label in targets}
+    target_train = {
+        label: min(totals[label], int(math.floor(totals[label] * targets[label] + 0.5)))
```

Ruff (0.15.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)